kava-stats.js

```diff
@@ -776,203 +759,140 @@ const setDisplayValueById = (cssId, value) => {
+    const swpMarketData = await mapCoinGeckoApiData(swpMarketDataJson)
+    siteData['marketData']['swp-a']['priceChangePercent'] = swpMarketData;
+    const prices = await mapPrices(denoms, pricefeedPrices.result);
+    siteData['prices'] = prices;
+    siteData['prices']['busd-b'] = siteData['prices']['busd-a'];
+    const swpPrice = await setSwpPrice(swpMarketDataJson);
+    siteData['prices']['swp-a'] = swpPrice;
+    const incentiveParamsData = await mapIncentiveParams(denoms, incentiveParamsJson.result.usdx_minting_reward_periods)
+    siteData['incentiveParamsData'] = incentiveParamsData;
+    const platformAmounts = await mapPlatformAmounts(totalCollateralJson.result, totalPrincipalJson.result); 
+    siteData['platformAmounts'] = platformAmounts;
+    const cdpParamsData = await mapCdpParams(denoms, cdpParamsJson.result);
+    siteData['cdpParamsData'] = cdpParamsData
+    const suppliedAmounts = mapSuppliedAmounts(denoms, suppliedAmountJson.result.value.coins);
+    siteData['suppliedAmounts'] = suppliedAmounts;
+    siteData['suppliedAmounts']['busd-b'] = siteData['suppliedAmounts']['busd-a'];
+    const totalSuppliedData = await mapDenomTotalSupplied(denoms, siteData)
+    siteData['totalSuppliedData'] = totalSuppliedData;
+    const supplyData = mapSuppliedAmounts(denoms, supplyTotalJson.result);
+    siteData['supplyData'] = supplyData;
+    const bep3SupplyData = await mapBep3Supplies(denoms, bep3SupplyJson.result);
+    siteData['bep3SupplyData'] = bep3SupplyData;
+    const bep3ParamsData = await mapBep3Params(denoms, bep3ParamsJson.result.asset_params, siteData);
+    siteData['bep3ParamsData'] = bep3ParamsData;
+    const defiCoinsSupply = await mapSupplyAndMarket(denoms, siteData)
+    siteData['defiCoinsSupply'] = defiCoinsSupply;
+    // set display values
+    await setTotalEarningsDisplayValues(denoms, siteData, cssIds)
+    await setPriceDisplayValues(denoms, siteData, cssIds)
+    await setPriceChangeDisplayValues(denoms, siteData, cssIds)
+    await setTotalBorrowedBorrowLimitAndLimitBarDisplayValues(denoms, siteData, cssIds)
+    await setRewardsApyDisplayValues(denoms, siteData, cssIds)
+    await setAssetLimitDisplayValues(denoms, siteData, cssIds)
+    await setAssetLimitUsdxDisplayValue(denoms, siteData, cssIds)
+    await setDenomTotalSuppliedDisplayValues(denoms, siteData, cssIds)
+    // denoms not needed here since totalSupplieData has already looped through the denoms
+    await setTotalAssetsSuppliedDisplayValue(siteData, cssIds)
+    await setTotalAssetsBorrowedDisplayValue(siteData, cssIds)
+    await setMarketCapDisplayValues(denoms, siteData, cssIds)
+    await setSupplyDisplayValues(denoms, siteData, cssIds);
+    await setBorrowApyDisplayValues(denoms, siteData, cssIds);
+    $(".metric-blur").css("background-color", "transparent")
+    $(".metric-blur").addClass('without-after');
+    $(".api-metric").css({"display": "block", "text-align": "center"})
```

Comment:
Removing the spaces makes it harder to see what has actually changed. In future PR's, please keep the structure consistent, unless you notice an error of course.